validate/validate_test.go

```diff
@@ -212,6 +212,38 @@ func TestCheckProcess(t *testing.T) {
 				Process: &rspec.Process{
 					Args: []string{"sh"},
 					Cwd:  "/",
+				},
+			},
+			platform: "windows",
+			expected: specerror.ProcCwdAbs,
+		},
+		{
+			val: rspec.Spec{
+				Version: "1.0.0",
+				Process: &rspec.Process{
+					Args: []string{"sh"},
+					Cwd:  "c:\\foo",
```

Comment:
Can we get a `windows` test using this `Cwd` with `expected: specerror.NonError` as well?

Reply:
updated.